doc/proxy-clients.rst

```diff
@@ -56,9 +112,13 @@ is available as the second parameter::
     $nginx = new Nginx($servers, 'my-cool-app.com');
 
 If you have configured NGINX to support purge requests at a separate location,
-supply that location to the class as the third parameter::
+call `setPurgeLocation()`::
+
+    use FOS\HttpCache\ProxyClient\Nginx;
+
+    $nginx = new Nginx($servers, $baseUri);
+    $nginx->setPurgeLocation(
```

Comment:
i think there should be an example param and closing bracket for setPurgeLocation